mappings/net/minecraft/network/encryption/SignedChatMessage.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_7471 net/minecraft/network/encryption/SignedChatMessage
```

Comment:
If this applies to any message type, remove `Chat` and call it `SignedMessage`

Reply:
This can technically be applied to any message types, but afaik in vanilla this is used for chat/chatlike command only.